### ISSUE TITLE
Codecov takes very long time to run

### ISSUE THREAD

**Issue by anddrzejb** [FortuneN/FineCodeCoverage#185]
### Installed product versions
- Visual Studio: 2019 Community 16.11.2 Preview 1.0
- This extension: [xample 1.1.141

### Description
I am a contributor to blazor component libraries [AntBlazor](https://github.com/ant-design-blazor/ant-design-blazor). The first time I added the extension, everything seemed to work out-of-the box. The problem is that any consecutive  run takes approx 7 minutes. I tried removing bin & obj folder in the test project, but nothing changed.  The tests themselves are running in 18secs.

### Steps to recreate
1. Pull branch from my [fork of AntBlazor](https://github.com/anddrzejb/ant-design-blazor/tree/testProjectWithAdapter).
2. Execute tests.
3. Make a change (for example comment out or remove any test file in folder `tests\AntDesign.Tests`)
4. Run tests.
5. Wait...

### Current behavior
Run between 7 and 8 minutes.  

### Expected behavior
Should run maybe few seconds longer than the tests.

**Comment by tonyhallett** [comment]
Please check the FCC Output Window Pane to see if that identifies your issue.


Note that FCC copies all files in the directory containing your test dll to a subdirectory.  Perhaps that is causing an issue
C:\Users\tonyh\source\repos\ant-design-blazor\tests\AntDesign.Tests\bin\Debug\net5\fine-code-coverage\build-output

**Comment by tonyhallett** [comment]
I compared coverage collection time ( using data collector ) of FCC against the copied dll vs dotnet test on the original and on my machine

![image](https://user-images.githubusercontent.com/11292998/132993002-c7a032b2-1883-4b28-8382-69151155219b.png)

The results are similar - about 3.5 minutes

Tests take 1s to run.

So coverage duration is not an FCC issue.  Although you may encounter issues due to the copying of the bin directory

**Comment by anddrzejb** [comment]
I just remembered we had some crazy times on running tests in github actions (with coverage collections) - 18~24minutes. Switching to tests in release configuration changed that to 3 minutes. I did the same now - and for my crappy machine I went down from 8 min to 2 min. Serious improvement, but still to long for serious work. I wonder if this has something to do with blazor testing framework - bUnit. But I guess this is not your problem. Thanks! BTW fantastic extension!!!
